install/infra/single-cluster/gcp/Makefile

```diff
@@ -7,10 +7,10 @@ init:
 	@terraform init
 
 touch-kubeconfig:
-	@touch kubeconfig
+	@touch kubeconfig ./dns-credentials.json
 
 cleanup-kubeconfig:
-	@rm kubeconfig
+	@rm kubeconfig ./dns-credentials.json
```

Comment:
Removed the `dns-credentials.json` file cleanup. Not necessary anymore. Both these files were added for `terraform plan` to succeed. Without a kubeconfig file, the cert manager and external dns stages wouldn't get planned since they depend on it.